docs/broker/filtering.md

```diff
@@ -66,16 +66,16 @@ expression languages.
 spec:
   filter:
     expression: >
-      ce.type == "com.github.pull.create" &&
-      ce.source == "https://github.com/knative/eventing/pulls/123"
```

Comment:
@mattmoor we don't want this. Any hints on how to prevent it?

Reply:
Use a different style of line wrapping?  e.g. `expression: |`